zebra-rpc/Cargo.toml

```diff
@@ -43,7 +43,7 @@ jsonrpc-http-server = "18.0.0"
 num_cpus = "1.14.0"
 
 # zebra-rpc needs the preserve_order feature in serde_json, which is a dependency of jsonrpc-core
-serde_json = { version = "1.0.89", features = ["preserve_order"] }
+serde_json = { version = "1.0.89", features = ["preserve_order", "arbitrary_precision"] }
```

Comment:
If some serde operations fail in weird ways with this feature, that's due to:
https://github.com/serde-rs/json/issues/505

Which is still open as:
https://github.com/serde-rs/serde/issues/1183

The fix for this is to stop serializing and deserializing `u128`s. We only need `u64`s for any feasible amount of work.